src/ui/hooks/recordings.ts

```diff
@@ -119,18 +147,23 @@ export function useGetOwnersAndCollaborators(recordingId: RecordingId) {
     console.error("Apollo error while getting owners and collaborators", error);
   }
 
-  const collaborators: CollaboratorDbData[] = data.collaborators;
-  const recording = data.recordings_by_pk;
+  const collaborators: CollaboratorDbData[] = data.recording.collaborators.edges.map(
```

Comment:
This needs a `?` on `collaborators` because the new API returns `null` if you aren't a collaborator/workspace member/owner, e.g. `data.recording.collaborators?.edges.map(`